.github/workflows/self-scheduled.yml

```
@@ -209,6 +209,13 @@ jobs:
         working-directory: /workspace/transformers
         run: git fetch && git checkout ${{ github.sha }}
 
+      - name: Re-compile DeepSpeed
+        working-directory: /workspace
+        run: |
+          rm -rf DeepSpeed
```

```suggestion
          pip install deepspeed
          rm -rf DeepSpeed
```

Let's add first a normal deepspeed install, which takes care of installing all the deps.

The manual pre-build seems to be no longer completely supported by pip and won't install any binary pip packages